packages/dds/tree/src/test/forestTestSuite.ts

```diff
@@ -764,6 +764,76 @@ export function testForest(config: ForestTestConfiguration): void {
 				assert.deepEqual(dependent.tokens.length, 1);
 			});
 		});
+
+		describe("When deleting the last node in the field", () => {
+			it("leaves an empty field", () => {
+				const forest = factory(new InMemoryStoredSchemaRepository(defaultSchemaPolicy));
+				const content: JsonableTree[] = [
```

Comment:
rather than using jsonable tree, since your data in in the json domain, you can simplify this, then below use singleJsonCursor to load it into the tree.

Reply:
also rather than createing an array at the root here then calling .map on it below, just make a single tree here then do `[singleJsonCursor(content)]` below since thats a little simpler